vulnerabilities/import_runner.py

```diff
@@ -70,17 +70,23 @@ def do_import(self, advisories) -> None:
             if advisory.date_imported:
                 continue
             logger.info(f"Processing advisory: {advisory!r}")
+            advisory_data = None
+            inferences = None
             try:
-                inferences = advisory_importer.get_inferences(
-                    advisory_data=advisory.to_advisory_data()
-                )
+                advisory_data = advisory.to_advisory_data()
+                inferences = advisory_importer.get_inferences(advisory_data=advisory_data)
                 process_inferences(
                     inferences=inferences,
                     advisory=advisory,
                     improver_name=importer_name,
                 )
-            except Exception as e:
-                logger.info(f"Failed to process advisory: {advisory!r} with error {e!r}")
+            except Exception:
+                from pprint import pformat
```

Comment:
@pombredanne Would it be better to move the `pprint` library import to the top of the code?